### ISSUE TITLE
Active Grains List

### ISSUE THREAD

**Issue by Dewaldf** [dotnet/orleans#1751]
Is there any way to get a list of all active grains. I understand this goes against the Orleans model, but will be used purely for a Dashboard to be able to visualise a Orleans Silo.

Currently we can get the count of active grains using the 

`IManagementGrain.GetTotalActivationCount().`

I managed to get the information using the following. But the methods and classes are internal, so there is no way to use this from an application.

```
var systemManagement = GrainClient.GrainFactory.GetGrain<IManagementGrain>(1);
var silos = GetSiloAddresses();
if (silos == null || silos.Count == 0) return;

var siloControl = GrainClient.InternalGrainFactory.GetSystemTarget<ISiloControl>(Constants.SiloControlId, silos[0]);
var grainStats = await siloControl.GetGrainStatistics();
```

**Comment by sergeybykov** [comment]
I think the right solution here is to add `GetGrainStatistics()` to `IManagementGrain` and implement it similarly to `GetSimpleGrainStatistics()`. Would you submit a PR for that?

**Comment by Dewaldf** [comment]
Will do.

**Comment by sergeybykov** [comment]
Resolved via #1772.
